loadgen/helpers.go

```diff
@@ -17,8 +18,8 @@ type distValueType interface {
 }
 
 type distribution[T distValueType] interface {
-	// Sample returns a random value from the distribution.
-	Sample() (T, bool)
+	// Sample returns a random value from the distribution using the provided seed.
+	Sample(seed int64) (T, bool)
```

Comment:
`Sample` now takes a seed.

Reply:
I'm thinking instead of a seed, it should take a rand.Rand. That way you can sample multiple distributions in sequence